templates/unoconv.service.j2

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Unoconv listener for document conversions
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/python3 /usr/bin/unoconv --listener -p 2002
```

Comment:
Is that port configured somewhere or is it the default?

Reply:
Yes, 2002 is the default port. I removed the `-p` flag.